### PR TITLE
Nothing really serious...

### PR DESCRIPTION
**Pull request by Ankit2kumar**

### CODE REVIEW

File: server/src/routes/admin.ts [hunk 1]
```diff
@@ -18,14 +18,14 @@ adminRouter.post("/signup", async (req: Request, res: Response) => {
...
     }
-
     const hashedPassword: string = await bcrypt.hash(password, 8);
     await prisma.admin.create({
       data: {
         email,
         hashedPassword: hashedPassword,
       },
-    });
+    })
```
Comment (somu-code) [inline server/src/routes/admin.ts:27]: What is the point of removing that semicolon?

File: server/src/routes/admin.ts [hunk 2]
```diff
@@ -18,14 +18,14 @@ adminRouter.post("/signup", async (req: Request, res: Response) => {
...
+
```
Comment (somu-code) [inline server/src/routes/admin.ts:28]: Why you need space here?
Just run "npm run format" before commit/push anything. Prettier will take care of it.

**Review by somu-code — CHANGES_REQUESTED** [review]
Run "npm run format" then push again.

File: server/src/routes/admin.ts (hunk 1 above)
Comment (Ankit2kumar) [reply server/src/routes/admin.ts:27]: Actually nothing, I was just seeing and trying something...

I didn't do it intentionally tbh. :)

Comment (somu-code) [reply server/src/routes/admin.ts:27]: Be serious man we are building an industry level project not a kidde project.
I might sound harsh but I am just giving you honest feedback hope you understand that part. I can merge anything and everything but if I do that you will not groww as a developer.

**Review by somu-code — APPROVED** [review]

**Pull request merged**